src/app/button/button.component.scss

```diff
@@ -0,0 +1,24 @@
+@import "src/styles/variables";
+@import "src/styles/maps";
+
+.button {
+  border: none;
+  background-color: $blue;
+  min-width: 60px;
+  display: flex;
+  flex-wrap: nowrap;
+  align-items: center;
+  gap: 10px;
+  border-radius: 4px;
+  padding: 5px 15px;
+  outline: none;
+  transition: 0.2s ease-in-out;
```

Comment:
Can you help me see where the transition is needed?

Reply:
for hover styles